plugins/lookup/aws_secret.py

```diff
@@ -225,9 +225,19 @@ def run(self, terms, variables=None, boto_profile=None, aws_profile=None,
                             secrets.update({secret['Name']: self.get_secret_value(secret['Name'], client,
                                                                                   on_missing=missing,
                                                                                   on_denied=denied)})
+
+                    # Manual pagination
+                    while 'NextToken' in response:
```

Comment:
@mandar242 Thank you for working on this. I guess you can use directly a paginator https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/secretsmanager.html#SecretsManager.Paginator.ListSecrets. In addition, could you please add some unit testing?